zap/src/main/java/org/zaproxy/zap/view/ContextIncludePanel.java

```diff
@@ -64,6 +75,56 @@ public static String getPanelName(Context context) {
         return getPanelName(context.getId());
     }
 
+    private File selectImportFile() {
+        JFileChooser chooser = new JFileChooser(Constant.getContextsDir());
+        File file = null;
+
+        int rc = chooser.showOpenDialog(View.getSingleton().getMainFrame());
+        if (rc == JFileChooser.APPROVE_OPTION) {
+            file = chooser.getSelectedFile();
+            if (file == null || !file.exists()) {
+                return null;
+            }
+            return file;
+        }
+        return null;
+    }
+
+    private void addImportButton() {
+        String IMPORT_BUTTON_LABEL = Constant.messages.getString("context.include.import.label");
+        JButton importButton = new JButton(IMPORT_BUTTON_LABEL);
```

Comment:
There doesn't seem to be any need for this intermediate storage and it isn't a CONSTANT.
```suggestion
        JButton importButton = new JButton(Constant.messages.getString("context.include.import.label"));
```

Reply:
Right, changed.